evm/src/cpu/kernel/opcodes.rs

```diff
@@ -35,6 +35,7 @@ pub(crate) fn get_opcode(mnemonic: &str) -> u8 {
         "SHR" => 0x1c,
         "SAR" => 0x1d,
         "KECCAK256" => 0x20,
+        "SHA3" => 0x20,
```

Comment:
I haven't been able to find out what's the consensus for the 0x20 mnemonic.

Sources using `SHA3`:
- Official Ethereum website https://ethereum.org/en/developers/docs/evm/opcodes/
- OpenEthereum
- evm.codes
- ethervm.io

Sources using `KECCAK256`:
- Yellow paper
- Geth

Reply:
I guess there was never a formal decision about it (EIP-1803 proposed the rename but it says stagnant), but it seems like `KECCAK256` is generally preferred, and it's just taking a while for all the `SHA3` uses to go away. The yellowpaper recently switched to `KECCAK256`, and that seems most authoritative IMO.